src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -223,7 +254,6 @@ public void getUsername_zeroId_returnsEmptyString() {
     Assert.assertEquals(actual, expected);
   }
 
-  @Test
```

Comment:
I don't think this was meant to be deleted?

Reply:
Ack!